app/routes/admin.js

```diff
@@ -0,0 +1,69 @@
+const router = require("express").Router();
+const mongoose = require("mongoose");
+const Request = require("../../models/request");
+const User = require("../../models/users");
+const Donor = require("../../models/donor");
+const bodyParser = require("body-parser");
+
+const authCheck = (req, res, next) => {
+  if (!req.user) {
+    res.redirect("/auth/google");
+  } else {
+    next();
+  }
+};
+
+router.get("/", authCheck, (req, res) => {
+  console.log(req.user);
+  Request.find().then(currentRequest => {
+    console.log(currentRequest);
+    var details = {
+      user_details: req.user,
```

Comment:
user_details to user, request_details to requests, no need of no_of_requests, can be calculated in frontend. and give to frontened like this res.render("admin", {...details})